src/store/index.js

```diff
@@ -126,7 +127,9 @@ export function setThemeInstance(instance) {
     bus.$emit('values-migrated')
   }
 
-  bus.$emit('saved-fields-set')
+  if (!options.fromSave) {
```

Comment:
@hcarneiro This function should not be concerned about what the caller is or what the scenario was.

Instead, it should open up a parameter for the caller to control whether the event should be emitted/suppressed. cc @squallstar